test/test_geometry.jl

```diff
@@ -120,5 +120,57 @@ datapath = joinpath(dirname(pathof(JUDI)))*"/../data/"
 
         @test compareGeometry(src_geometry, src_geometry)
         @test compareGeometry(rec_geometry, rec_geometry)
+
```

Comment:
Hi @kerim371 , you can find how testing works here https://github.com/slimgroup/JUDI.jl#testing. I've made the CI tests available for you.

I gave it a 2nd look. If possible, can you actually use https://github.com/slimgroup/JUDI.jl/blob/91f1e600539094874c1486ce75173ccdee1bd3bd/test/utils.jl#L135 and https://github.com/slimgroup/JUDI.jl/blob/91f1e600539094874c1486ce75173ccdee1bd3bd/test/utils.jl#L137 and https://github.com/slimgroup/JUDI.jl/blob/91f1e600539094874c1486ce75173ccdee1bd3bd/test/utils.jl#L146 in the test? This will make the addition much shorter.

In terms of where to put, I think putting it here https://github.com/slimgroup/JUDI.jl/blob/91f1e600539094874c1486ce75173ccdee1bd3bd/test/test_issues.jl looks better to me.

Reply:
thank you! when discovering new things (testing) it always takes some time.
It is late night, tomorrow I will try to fix

Reply:
fixed